manifests/charts/istiod-remote/templates/role.yaml

```diff
@@ -25,4 +25,8 @@ rules:
   resources: ["configmaps"]
   verbs: ["delete"]
 
+# For gateway deployment controller
```

Comment:
Is Istiod doing distributed master elections for remote clusters ? Doesn't seem very safe.

Is this for the strange mode where a central Istiod manages multiple clusters ? Maybe we want to separate this case, it requires granting a lot of permissions.

Reply:
This is just Istiod running in cluster X using Cluster Y as config cluster. Probably we shouldn't have had its own chart for this, but we did... the good news is its not part of our Helm Beta API surface (not published)

Note there is some `make gen` that just syncs these files